atlp-rwanda/atlp-pulse-mobile
# ft(#03): Login Implementation

### What does this PR do?

This PR implements the login flow where the user first enters the organization name before proceeding to log in with their email and password. It also integrates the backend using Apollo and GraphQL for authentication.

### Description of Task to be completed?

- Add functionality for users to input the organization name first.
- Implement logic to verify the organization before allowing the user to proceed to enter email and password.
- Establish Apollo and GraphQL integration with the backend for authentication processes.
- Ensure the login flow handles both organization and user credential verification in sequence.

### How should this be manually tested?

- Start the application.
- On the login screen, input a valid organization name.
- After successful organization verification, proceed to input the user's email and password.
- Ensure successful login after submitting valid credentials.
- Check the network to confirm backend integration with Apollo and GraphQL for authentication.

### Screenshots: 

<div style="display: flex; flex-wrap: wrap;">
    <img src="https://github.com/user-attachments/assets/3dba1cad-fc09-43b2-af45-31cbdb4fbb14" alt="Screenshot 1" height="480px">
    <img src="https://github.com/user-attachments/assets/f83233c1-bf4e-4b7b-bf8c-136ee9a3f95af" alt="Screenshot 2" height="480px">
        <img src="https://github.com/user-attachments/assets/f842e352-397a-4d08-8656-3a18352bd90e" alt="Screenshot 1" height="480px">
    <img src="https://github.com/user-attachments/assets/63567453-30da-4240-ae95-a9336b915012" alt="Screenshot 2" height="480px">
</div>
